site/styles/_availability-tags.scss

```diff
@@ -71,3 +71,25 @@ s72-plan-label {
   font-size: 12px;
   text-align: center;
 }
+
+s72-live-lozenge {
+  .live-lozenge {
+    background-color: var(--live-lozenge-background-color);
```

Comment:
We can utilise `backdrop-filter: blur(...)` and rgba to get the blur effect in the designs

Reply:
We made a decision a while back to not use opacity in core-template because of the potential accessibility issues with client colours + client background images.

Reply:
Can we confirm with product?

Reply:
Switched to blur though it doesn't look very good IMO.